### ISSUE TITLE
Builds fail for 1.53

### ISSUE THREAD

**Issue by jacksongoode** [VSCodium/vscodium#629]
Just wanted to add some visibility

**Comment by hazarek** [comment]
When will 1.53 be ready?

**Comment by stripedpajamas** [comment]
Ok, looks like one of our telemetry patches doesn't match anymore. 

```
 ++ patch -u src/vs/platform/update/electron-main/updateService.win32.ts -i ../patches/update-cache-path.patch
patching file src/vs/platform/update/electron-main/updateService.win32.ts
Hunk #1 FAILED at 55.
1 out of 1 hunk FAILED -- saving rejects to file src/vs/platform/update/electron-main/updateService.win32.ts.rej
```

Will need to see where they moved things around to.

**Comment by stripedpajamas** [comment]
Looks like @sscotth is ahead of me with #628. Will see if errors remain after merging.

**Comment by stripedpajamas** [comment]
Seems like we're being rate limited during installation of node deps, by GitHub. Setting `GITHUB_TOKEN` env var is supposed to get around this, but we're already doing that. I'll re-run the workflows in a couple minutes to see if the rate limit has been reset.

**Comment by sscotth** [comment]
The linux builds have a couple other issues as well.

https://github.com/microsoft/vscode/issues/111299

**Comment by jacksongoode** [comment]
Closed with 96d79aeb70e6931f62e563e4cdb63005b32ea95e
